stubs/Markdown/markdown/extensions/codehilite.pyi

```diff
@@ -18,6 +18,7 @@ class CodeHilite:
     tab_length: Any
     hl_lines: Any
     use_pygments: Any
+    options: Dict[str, Any]
     def __init__(
```

Comment:
Seems like we should also add `**kwargs: Any` to `__init__`. The source code looks like this:

```
    def __init__(self, src, **options):
        self.src = src
        self.lang = options.pop('lang', None)
        self.guess_lang = options.pop('guess_lang', True)
        ...
        self.options = options
```

Also, in `__init__`, it's missing `*` between `src` and `linenums` to mark the rest of the arguments as keyword-only.

Reply:
Good find!
I guess we only need to add `*` between `src` and `linenums`.